mmdet/models/detectors/single_stage.py

```diff
@@ -160,7 +160,14 @@ def onnx_export(self, img, img_metas):
         # `CornerNet` and `CentripetalNet`, which 'pad_shape' is used
         # for inference
         img_metas[0]['pad_shape_for_onnx'] = img_shape
-        # TODO:move all onnx related code in bbox_head to onnx_export function
-        det_bboxes, det_labels = self.bbox_head.get_bboxes(*outs, img_metas)
+        if hasattr(self.bbox_head, 'onnx_export'):
+            if len(outs) == 2:
+                # add dummy score_facotr
+                outs = (*outs, None)
+            det_bboxes, det_labels = self.bbox_head.onnx_export(
+                *outs, img_metas)
+        else:
+            det_bboxes, det_labels = self.bbox_head.get_bboxes(
```

Comment:
do we need to raise a warning or it is expect to run with get_bboxes

Reply:
I will add it